src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/ForwardCancellationTokenToAsyncMethods.Fixer.cs

```diff
@@ -0,0 +1,126 @@
+            {
+                return;
+            }
+
+            SemanticModel model = await doc.GetSemanticModelAsync(ct).ConfigureAwait(false);
+            if (!(model.GetOperation(node, ct) is IInvocationOperation invocation))
+            {
+                return;
+            }
+
+            if (!TryGetAncestorDeclarationCancellationTokenParameterName(model, node, out string? parameterName))
+            {
+                return;
+            }
+
+            string title = MicrosoftNetCoreAnalyzersResources.ForwardCancellationTokenToAsyncMethodsTitle;
+
+            Task<Document> createChangedDocument(CancellationToken _) => FixInvocation(doc, root, invocation, parameterName!);
+
+            context.RegisterCodeFix(
+                new MyCodeAction(
+                    title: title,
+                    createChangedDocument,
+                    equivalenceKey: title),
+                context.Diagnostics);
+        }
+
+        private static Task<Document> FixInvocation(Document doc, SyntaxNode root, IInvocationOperation invocation, string cancellationTokenParamenterName)
+        {
+            SyntaxGenerator generator = SyntaxGenerator.GetGenerator(doc);
+
+            // Create a new ct argument to pass to the invocation, using the ancestor method parameter name
+            SyntaxNode cancellationTokenIdentifier = generator.IdentifierName(cancellationTokenParamenterName);
+            SyntaxNode cancellationTokenNode = generator.Argument(cancellationTokenIdentifier);
+
+            // Pass the same arguments and add the ct
+            List<SyntaxNode> newArguments = new List<SyntaxNode>();
+            foreach (IArgumentOperation argument in invocation.Arguments)
+            {
+                if (argument.Parameter.Type.Name != CancellationTokenName || !argument.IsImplicit)
+                {
+                    newArguments.Add(argument.Syntax);
+                }
+            }
+            newArguments.Add(cancellationTokenNode);
+
+
+            SyntaxNode newInvocation;
+            // The instance is null when calling a static method from another type
+            if (invocation.Instance == null)
+            {
+                SyntaxNode staticType = generator.TypeExpressionForStaticMemberAccess(invocation.TargetMethod.ContainingType);
+                newInvocation = generator.MemberAccessExpression(staticType, invocation.TargetMethod.Name);
+            }
+            // The instance is implicit when calling a method from the same type, call the method directly
+            else if (invocation.Instance.IsImplicit)
+            {
+                newInvocation = invocation.GetInstance(); // GetInstance will return the method name
+            }
+            // Calling a method from an object, we must include the instance variable name
+            else
+            {
+                newInvocation = generator.MemberAccessExpression(invocation.GetInstance(), invocation.TargetMethod.Name);
+            }
```

Comment:
The CI failed to build these lines, while I had no problems in my local machine.